ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/sflow_settings.schema.yml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  sflow_settings:
+    documentation_options:
+      table: management-sflow-settings
+    type: dict
+    description: |-
+      sFlow settings.
+      The sFlow process will only be configured if any interface is enabled for sFlow.
+      For default enabling of sFlow for various interface types across the fabric see "fabric_sflow"
+    keys:
+      destinations:
+        type: list
+        items:
+          type: dict
+          keys:
+            destination:
+              type: str
+              required: true
+              description: |-
+                sFlow destination IP address.
```

Comment:
Since this is a string does it make sense to specify this could be an IPv4 or IPv6 address? Or even an fqdn?

Reply:
Correct. I have updated to "name or IP address" like we have elsewhere.